Python/ListComprehensions.py

```diff
@@ -55,4 +55,31 @@
     z = int(input())
     n = int(input())
 
-print([[i,j,k] for i in range(x+1) for j in range(y+1) for k in range(z+1) if(i+j+k)!=n])
\ No newline at end of file
+print([[i,j,k] for i in range(x+1) for j in range(y+1) for k in range(z+1) if(i+j+k)!=n])
+
+
```

Comment:
in List comprehension how to use the if else if ladder so that we can use the inline approach and get our code readable properly